integration

```diff
@@ -320,6 +320,7 @@
   "evantaur/seiverkot-consumption",
   "evilmarty/mjpeg-timelapse",
   "exKAjFASH/media_player.elkoep_lara",
+  "ExPeacer/ha_openai_response",
```

Comment:
<https://github.com/ExPeacer/ha_openai_response/blob/v1.2.0/hacs.json#L4> Does not match release asset.